projects/core/src/lib/talks/talk-item/talk-item.component.html

```diff
@@ -0,0 +1,35 @@
+<div class="talks-item" *ngFor="let talk of talks; let i = index">
```

Comment:
Please, use `jsmu` prefix for each element, like jsmu-talk-item

Reply:
You mean rewrite all classes name with prefix -jsmu? i ask because in another merged branch we didn't do it

Reply:
Yes, now we should align with strong BEM rules.
We can discuss it on meeting in details.